dist/js/ui-components.js

```diff
@@ -301,6 +341,7 @@
 	        this.$window = $window;
 	        this.$location = $location;
 	        this.$sce = $sce;
+	        console.log(this.toolbarItems);
```

Comment:
Yeah removed the console.log but, forgot to rebuild the project before pushing. Good catch.